test/functional/sqlsrv/test_connectionOption.phpt

```diff
@@ -11,7 +11,7 @@ Test MultipleActiveResultSets connection setting off
 
     require( 'MsCommon.inc' );
 
-    $conn = ConnectSpecial(array( 'MultipleActiveResultSets' => false ));
+    $conn = Connect(array( 'MultipleActiveResultSets' => false ));
```

Comment:
Connect() basically merges the given array to the existing options and ConnectSpecial() makes sure uid, pwd and database are set using the defaults if not already defined